test/conf/nginx.conf

```diff
@@ -153,6 +153,46 @@ http {
         }
     }
 
+    server {
+        listen 58089 ssl;
+        server_name localhost;
+        ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+        ssl_ciphers         HIGH:!aNULL:!MD5;
+        ssl_certificate     __NGXDOCROOT__/dummy.crt;
+        ssl_certificate_key __NGXDOCROOT__/dummy.key;
+        ssl_verify_client on;
+        ssl_client_certificate __NGXDOCROOT__/dummy.crt;
+
+        mruby_ssl_verify_client_handler_code 'Nginx::SSL.reject_client';
+    }
+
+    server {
+        listen 58090 ssl;
+        server_name localhost;
+        ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+        ssl_ciphers         HIGH:!aNULL:!MD5;
+        ssl_certificate     __NGXDOCROOT__/dummy.crt;
+        ssl_certificate_key __NGXDOCROOT__/dummy.key;
+        ssl_verify_client on;
+        ssl_client_certificate __NGXDOCROOT__/dummy.crt;
+
+        mruby_ssl_verify_client_handler_code 'Nginx::SSL.accept_client';
+    }
+
+    server {
+        listen 58091 ssl;
+        server_name localhost;
+        ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+        ssl_ciphers         HIGH:!aNULL:!MD5;
+        ssl_certificate     __NGXDOCROOT__/dummy.crt;
+        ssl_certificate_key __NGXDOCROOT__/dummy.key;
+        ssl_verify_client on;
+        ssl_client_certificate __NGXDOCROOT__/dummy.crt;
+
+        mruby_ssl_handshake_handler_code 'class Test4SSLHandler; end';
+        mruby_ssl_verify_client_handler_code 'Test4SSLHandler';
```

Comment:
2. Please change to a test that performs unique processing in each Directives, not a simple test that just defines the class. For example, servername is acquired with ssl_handler, and verify_handler performs verify of its servername and certificate etc.

Reply:
`openssl s_client` not working properly I think. I left a message in the conversation.